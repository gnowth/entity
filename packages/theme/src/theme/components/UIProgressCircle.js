import { css } from 'styled-components';

export const component_uiProgressCircle_processing = {
  margin: '2rem auto',
  css: css`
    display: block;
  `,
};

export const component_uiProgressCircle_processingInline = {};
