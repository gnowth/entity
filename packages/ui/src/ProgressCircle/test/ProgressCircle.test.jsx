import renderer from 'react-test-renderer';
import React from 'react';

import UIProgressCircle from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <UIProgressCircle />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
