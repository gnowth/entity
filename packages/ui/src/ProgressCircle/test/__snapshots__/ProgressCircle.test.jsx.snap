// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c2 {
  -webkit-animation-name: bvvTNa;
  animation-name: bvvTNa;
  fill: transparent;
  stroke: black;
  stroke-dasharray: 251.32741228718345px;
  stroke-width: 20%;
  -webkit-transition: stroke-dashoffset 225ms linear;
  transition: stroke-dashoffset 225ms linear;
  -webkit-transform-box: view-box;
  -ms-transform-box: view-box;
  transform-box: view-box;
  -webkit-transform-origin: center;
  -ms-transform-origin: center;
  transform-origin: center;
  -webkit-animation-duration: 4s;
  animation-duration: 4s;
  -webkit-animation-iteration-count: infinite;
  animation-iteration-count: infinite;
  -webkit-animation-timing-function: cubic-bezier(0.35,0,0.25,1);
  animation-timing-function: cubic-bezier(0.35,0,0.25,1);
  -webkit-transition-property: stroke;
  transition-property: stroke;
}

.c1 {
  height: 100%;
  left: 0;
  top: 0;
  overflow: visible;
  position: absolute;
  -webkit-transform: rotate(-90deg);
  -ms-transform: rotate(-90deg);
  transform: rotate(-90deg);
  -webkit-transform-origin: center;
  -ms-transform-origin: center;
  transform-origin: center;
  width: 100%;
  -webkit-animation: eXzyYY 2s linear infinite;
  animation: eXzyYY 2s linear infinite;
}

.c0 {
  display: inline-block;
  height: 32px;
  position: relative;
  width: 32px;
}

<div
  className="c0"
  size="32px"
  value={null}
>
  <svg
    className="c1"
    focusable="false"
    preserveAspectRatio="xMidYMid meet"
    value={null}
    viewBox="0 0 100 100"
  >
    <circle
      className="c2"
      cx="50%"
      cy="50%"
      r="40"
      value={null}
    />
  </svg>
</div>
`;
